hazelcast/src/main/java/com/hazelcast/jet/Job.java

```diff
@@ -150,6 +150,15 @@ default String getName() {
     @Nonnull
     JobStatus getStatus();
 
+    /**
+     * Returns true, if the job is user-cancelled. Returns false, if it
+     * completed normally or failed due to another error.
```

Comment:
they will not have `userCancelled` field in `JobResult` so the default will be false. There is no robust method of detecting user-cancellation for pre-5.3 jobs. We cannot use exception because `CancellationException` is used also in some failures. Should I document behavior for pre-5.3 jobs?

Reply:
I would add a line about it here.

Reply:
added